ng/src/web/components/form/multiselect.js

```diff
@@ -0,0 +1,263 @@
+      // reset search term
+      search: '',
+    });
+  }
+
+  handleRemoveItem(item) {
+    const copy = [...this.state.selectedItems];
+    const index = copy.findIndex(elem => elem === item);
+    copy.splice(index, 1);
+    this.setState({
+      selectedItems: copy,
+    });
+  }
+
+  componentWillReceiveProps(nextProps) {
+    const {value} = nextProps;
+    if (arrays_equal(value, this.state.selectedItems)) {
+      this.setState({selectedItems: value});
+    }
+  }
+
+  renderItem(value, items, selectedItem) {
+    const item = items.find(i => i.value === value);
+    const itemLabel = is_defined(item) ? item.label : value;
+
+    return (
+      <MultiSelectedValue
+        title={itemLabel}
+        key={value}
+      >
+        <DeleteButton
+          onClick={() => this.handleRemoveItem(value)}
+        >
+          × {/* Javascript unicode: \u00D7 */}
+        </DeleteButton> {itemLabel}
+      </MultiSelectedValue>
+    );
+  }
+
+  render() {
+    let {
+      children,
+      className,
+      disabled = false,
+      items,
+      menuPosition,
+    } = this.props;
+    const {
+      search,
+      selectedItems,
+    } = this.state;
+
+    if (!is_defined(items)) {
+      items = option_items(children);
+    }
+
+    const displayedItems = items.filter(case_insensitive_filter(search));
+    return (
+      <Downshift
+        selectedItem={selectedItems}
+        onChange={this.handleChange}
+        onSelect={this.handleSelect}
+        render={({
+          getButtonProps,
+          getInputProps,
+          getItemProps,
+          getRootProps,
+          highlightedIndex,
+          inputValue,
+          isOpen,
+          openMenu,
+          selectedItem,
+        }) => {
+          return (
+            <SelectContainer
+              {...getRootProps({refKey: 'innerRef'})}
+              className={className}
+              flex="column"
+            >
+              <Box
+                isOpen={isOpen}
+              >
+                <Layout grow="1" wrap>
+                  {!is_empty(selectedItem) ? // eslint-disable-line no-negated-condition
+                    selectedItem.map(
+                      item => this.renderItem(item, items, selectedItem)
+                    ) :
+                      null
+                  }
+                </Layout>
+                <ArrowButton
+                  {...getButtonProps({
+                    disabled,
+                  })}
+                >
+                  <ArrowIcon down={isOpen}/>
+                </ArrowButton>
+              </Box>
+              {isOpen && items.length > 0 && !disabled &&
+                <Menu position={menuPosition}>
+                  <Input
+                    {...getInputProps({
+                      value: search,
+                      onChange: this.handleSearch,
+                    })}
+                    disabled={disabled}
+                    innerRef={ref => this.input = ref}
```

Comment:
Should be obsolete in MultiSelect. it was necessary to set the focus to the input when opening the menu.